### PR TITLE
[OCSI] add new osci author kshitijalwadhi

### PR DESCRIPTION
**Pull request by kshitijalwadhi**
### Description
Added author kshitijalwadhi for OCSI as per the guide.

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/osci_2023/kshitijalwadhi.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Kshitij Alwadhi
+short_name: Kshitij
+linkedin: kshitijalwadhi
+github: kshitijalwadhi
+---
+
+**<Kshitij Alwadhi>** <First year MSCS grad at Purdue University>
```
Comment (joshuarrrr) [inline _authors/osci_2023/kshitijalwadhi.markdown:8]: Please remove the `<>` brackets:

```suggestion
**Kshitij Alwadhi** First year MSCS grad at Purdue University
```

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
Just remove the brackets - everything else looks good.

**Comment by dtaivpp** [comment]
@kshitijalwadhi once you address Josh's comments we should be good to go. Thank you!

File: _authors/osci_2023/kshitijalwadhi.markdown (hunk 1 above)
Comment (kshitijalwadhi) [reply _authors/osci_2023/kshitijalwadhi.markdown]: Thanks.
Addressed in the following commit.

**Comment by dtaivpp** [comment]
Closing as still missing DCO and we are past deadline

**Pull request closed without merging**